backend/open_webui/utils/telemetry/setup.py

```
@@ -1,23 +1,101 @@
+logging.getLogger("opentelemetry.instrumentation.asgi").setLevel(logging.DEBUG)
+
+# Also set up debug logging for our own telemetry modules
+logging.getLogger("open_webui.utils.telemetry").setLevel(logging.DEBUG)
+
+logger = logging.getLogger(__name__)
+
+
+def test_otlp_connection(exporter):
+    """Test OTLP connection by attempting to create and export a test span"""
+    try:
+        import time
```

Ruff (0.12.2)

`time` imported but unused

Remove unused import: `time`

(F401)